### ISSUE TITLE
JSON output not working for [[String: Any]]

### ISSUE THREAD

**Issue by svanimpe** [elliottminns/blackfish#36]
I have the following controller set up:

```
class StudentController: Controller {

  private var students: [Student]

  init(students: [Student]) {
    self.students = students
  }

  func routes(router: Router) {
    router.get("/", handler: list)
  }

  private func list(request: Request, response: Response) {
    response.send(json: students.map { $0.toJSON() })
  }
}
```

Method toJSON in class Student is as follows:

```
func toJSON() -> [String: Any] {
    var json: [String: Any] = [:]
    json["id"] = id // Int
    json["name"] = name // String 
    json["enrollments"] = enrollments.map { $0.id } // [Int]
    return json
}
```

Trying to GET this resource returns an empty string.
In the console, Blackfish outputs the following:

```
BlackFishTest listening on port 3000
[["id": 1, "name": "Steven", "enrollments": []], ["id": 2, "name": "Ray", "enrollments": []]]
Mirror for Array<Dictionary<String, protocol<>>>
```

What's wrong here?

I'm running the February 8 build on Ubuntu.

**Comment by elliottminns** [comment]
Thanks for raising this. 

I have a feeling it's an issue with `Any`. There seems to be a bug within swift that doesn't properly handle it. Let me have a look and see what I can find and whether I have a work around.

I intend to add a better JSON parser to the project this weekend that should _hopefully_ clear up these issues.

**Comment by elliottminns** [comment]
From what I can see, it's the that theres a array of mixed types not being able to be casted to [Any] properly. 

I think i'll need to do the same as I did for the MongoDB data wrapper and add in something to convert these explicitly.

I'll get working on that tonight and let you know when I have something running.

**Comment by elliottminns** [comment]
I've got this fixed for the moment in PR #37 .

I shall merge this in under tag 0.1.12

Let me know if you encounter any further JSON issues.
